x-pack/plugins/watcher/public/application/sections/watch_edit/watch_edit_actions.ts

```diff
@@ -67,7 +67,7 @@ export async function saveWatch(watch: BaseWatch, toasts: ToastsSetup): Promise<
     await createWatch(watch);
     toasts.addSuccess(
       i18n.translate('xpack.watcher.sections.watchEdit.json.saveSuccessNotificationText', {
-        defaultMessage: "Saved '{watchDisplayName}'",
+        defaultMessage: (watch.isNew ? 'Created' : 'Saved') + ' {watchDisplayName}',
```

Comment:
I don't think this is valid; you'll want to have two separate translations instead.

For example:

```
      watch.isNew
        ? i18n.translate('xpack.watcher.sections.watchEdit.json.createSuccessNotificationText', {
            defaultMessage: "Created '{watchDisplayName}'",
            values: {
              watchDisplayName: watch.displayName,
            },
          })
        : i18n.translate('xpack.watcher.sections.watchEdit.json.saveSuccessNotificationText', {
            defaultMessage: "Saved '{watchDisplayName}'",
            values: {
              watchDisplayName: watch.displayName,
            },
          })
```

You can read more about i18n in Kibana [here](https://github.com/elastic/kibana/blob/master/packages/kbn-i18n/README.md) and [here](https://github.com/elastic/kibana/blob/master/packages/kbn-i18n/GUIDELINE.md).